modules/deploy/pages/deployment-option/cloud/whats-new-cloud.adoc

```diff
@@ -0,0 +1,22 @@
+= What's New in Redpanda Cloud
+:description: Summary of new features in Redpanada Cloud releases.
+
+This topic lists new features added in Redpanda Cloud. 
+
+== November 2023
+
+==== Optimized BYOC tiers support ARM-based AWS Graviton processors
+
+BYOC clusters now support ARM-based AWS Graviton processors on Amazon EC2. This allows decreased VM costs and increased partition support.
+
+==== Iceberg Sink connector
+
+With the xref:deploy:deployment-option/cloud/managed-connectors/create-iceberg-sink-connector.adoc[managed connector for Apache Iceberg], you can write data into Iceberg tables. This enables integration with the data lake ecosystem and efficient data management for complex analytics.
+
+==== Schema Registry management
```

Comment:
Are there updated docs for SR? If so, should link to them.

Reply:
thanks, I added a link to the new doc!